p2p/discovery/getaddress.go

```diff
@@ -56,6 +57,12 @@ func (p *protocol) GetAddresses(server p2pcrypto.PublicKey) ([]*node.NodeInfo, e
 		//todo: check that we're not pass max results ?
 		if err != nil {
 			p.logger.Warning("could not deserialize bytes to NodeInfo, skipping packet err=", err)
+			return
+		}
+
+		if len(nodes) > getAddrMax {
+			p.logger.Warning("Got a too big addresses messages from %v. skipping", server.String())
```

Comment:
Log the expected and the actual
Also, rephrase the message, it is not clear what you mean